diff --git a/lib/utils.js b/lib/utils.js
index 50264bdf08dd24831851ebf47a65d87966a90a18..8ea79f8981906b3af0fe7e9835e732c4f7030779 100644
--- a/lib/utils.js
+++ b/lib/utils.js
@@ -87,6 +87,8 @@ function spawnProcess(command, args, options) {
       reject(err);
     });
     child.on('close', exitCode => {
+      console.log('!!! STDOUT !!!', stdout);
+
       if (exitCode !== 0) {
         reject(new SpawnError(`${command} ${_.join(args, ' ')} failed with code ${exitCode}`, stdout, stderr));
       } else {
